apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -19,7 +18,8 @@ export default function ListBox({
   height,
   width,
   listLayout = 'vertical',
-  rangeSelect = true,
+  rangeSelect = false,
+  checkboxes = false,
```

Comment:
I am unsure if I think we should enable rangeSelect by default (we should atleast do it for the next major version)
but if not. do enable it for nebula serve.

Reply:
Ok, I can turn it off for now